views/js/qtiCreator/widgets/interactions/helpers/shapeEditor.js

```diff
@@ -192,13 +192,15 @@ define([
                         content : parseInt(bbox.width, 10) + ' x ' + parseInt(bbox.height, 10)
                     });
 
+                    function setPointIndex(point, index){
+                        if(point.length === 3 && point[1] === this.attr('cx') && point[2] === this.attr('cy')){
+                            this.pointIndex = index;
+                            return false;
+                        }
+                    }
+
                     if(self.shape.type === 'path'){
-                       _.forEach(self.shape.attr('path'), function(point, index){
-                           if(point.length === 3 && point[1] === this.attr('cx') && point[2] === this.attr('cy')){
-                                this.pointIndex = index;
-                                return false;
-                           }
-                       }, handler);
+                        self.shape.attr('path').forEach(setPointIndex.bind(handler));
```

Comment:
**suggestion:** No need to bind the function. `array.forEach()` already have auto binding, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#syntax
```suggestion
                        self.shape.attr('path').forEach(setPointIndex, handler);
```

Reply:
done